sormas-ui/src/main/java/de/symeda/sormas/ui/reports/aggregate/AggregateReportsEditLayout.java

```diff
@@ -251,10 +258,16 @@ public AggregateReportsEditLayout(Window window, AggregateReportCriteria criteri
 		addComponent(legend);
 		legend.addStyleName(CssStyles.VSPACE_TOP_1);
 
-		editForms.sort((e1, e2) -> e1.getDisease().compareTo(e2.getDisease()));
-		for (AggregateReportEditForm editForm : editForms) {
-			addComponent(editForm);
-		}
+		editForms.stream()
+			.sorted(
+				Comparator.comparing(AggregateReportEditForm::getDisease, Comparator.nullsFirst(Comparator.comparing(Disease::toString)))
+					.thenComparing(
```

Comment:
I think AgeGroupUtils.getComparator can be used here.

Reply:
changed